### PR TITLE
ENH changed color map + rm algo param from grid search

### PR DESCRIPTION
**Pull request by QB3**
x weird cmap pb in enet

### CODE REVIEW

File: examples/plot_held_out_enet.py [hunk 1]
```diff
@@ -127,7 +127,7 @@
 ax.scatter(
     monitor.alphas[:, 0] / alpha_max,
     monitor.alphas[:, 1] / alpha_max,
-    s=50, cmap=cmap, c=c,
+    s=50, color=cmap(c),  zorder=len(c),
```
Comment (mathurinm) [inline examples/plot_held_out_enet.py:130]: zorder= high number  should be enough (it's so that it appears on top of other plotted stuff)

**Comment by mathurinm** [comment]
while there's a PR, in grid_search there's an algo instanciated which can be removed

**Comment by QB3** [comment]
However it seems that the transparency has been broken:
https://658-252772479-gh.circle-artifacts.com/0/dev/auto_examples/plot_held_out_enet.html#sphx-glr-auto-examples-plot-held-out-enet-py

**Comment by QB3** [comment]
> while there's a PR, in grid_search there's an algo instanciated which can be removed

Do you want to instantiate it inside? in an hidden way?

**Comment by mathurinm** [comment]
Hm I thought that it was harmless to delete algo, but missed that grid_search needed it. Why ? It's not used inside the `grid_search` function

**Comment by QB3** [comment]
> Hm I thought that it was harmless to delete algo, but missed that grid_search needed it. Why ? It's not used inside the grid_search function

Humm, it seems that it is simpler that what I thought indeed.
I will remove this `algo`

**Comment by QB3** [comment]
good for you @agramfort ?

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
thx @QB3 and @mathurinm

**Pull request merged**